dev/run-dev-server.sh

```diff
@@ -1,32 +1,28 @@
 #!/bin/bash
 set -e
 
-log_file="outputs/mlflow-server.log"
```

Comment:
Nit: now the mlflow server log and yarn server log will be mixed and print to screen. Is it an issue ?

Reply:
I don't think it is an issue.

Reply:
I stopped logging to a file to make it easier to check hot reloading is triggered.

Reply:
Having both server and yarn logs in the same terminal might be confusing, but not a big issue.